profiles/permissions.easy

```diff
@@ -315,3 +315,7 @@
 
 # nvidia-modprobe (bsc#1230950)
 /usr/bin/nvidia-modprobe                                root:root 4755
+
+# OpenSMTPD (bsc#1247781)
+/usr/libexec/opensmtpd/lockspool                        root:root 4755
+/usr/sbin/smtpctl                                       root:_smtpq 2555
```

Comment:
Why is the smtpctl binary `2555` in easy, but `2755` in secure?

Reply:
Right, I forgot to sync this. Upstream sets this to `2555` but I don't think this makes much sense.